lib/matplotlib/pyplot.py

```diff
@@ -154,6 +155,13 @@ def post_execute():
 
             _IP_REGISTERED = post_execute
             _INSTALL_FIG_OBSERVER = False
+
+            # trigger IPython's eventloop integration, if available
+            from IPython.core.pylabtools import backend2gui
+
+            ipython_gui_name = backend2gui.get(get_backend())
```

Comment:
Nothing. Since it's a `get` from a dict, gui_name will be None and it will take no further action, which can be interpreted as either:

1. IPython doesn't know how to integrate with the eventloop for the given backend, or
2. it doesn't need to (e.g. Agg)